lib/dictionary

```diff
@@ -146,6 +146,7 @@ freenode
 Frogger
 Fu
 funder
+Funders
```

Comment:
Yeah, you're totally right. Let me fix that.

Sent from my mobile

> On May 20, 2016, at 5:05 PM, JESSICA <notifications@github.com> wrote:
> 
> In lib/dictionary:
> 
> > @@ -146,6 +146,7 @@ freenode
> >  Frogger
> >  Fu
> >  funder
> > +Funders
> shouldnt this be funders lowercase
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>